pennylane_lightning/lightning_tensor/lightning_tensor.py

```diff
@@ -147,7 +171,15 @@ def stopping_condition(op: Operator) -> bool:
     # These thresholds are adapted from `lightning_base.py`
     # To avoid building matrices beyond the given thresholds.
     # This should reduce runtime overheads for larger systems.
-    return op.has_matrix and len(op.wires) <= 2 and op.name in _operations
+    if isinstance(op, qml.QFT):
+        return len(op.wires) < 10
+    if isinstance(op, qml.GroverOperator):
+        return len(op.wires) < 13
```

Comment:
Did you test these limits? Because the behaviour can be very different from `lightning.qubit`. Especially in the MPS context.

Reply:
Not really. Good ponits!